src/mds/ScrubStack.h

```diff
@@ -203,15 +203,15 @@ class ScrubStack {
    * @param added_children set to true if we pushed some of our children
    * @param done set to true if we started to do final scrub
    */
-  void scrub_dir_inode(CInode *in, bool *added_children, bool *done);
+  void scrub_dir_inode(CInode *in, bool *added_children, bool *done, bool is_eval_stray = false);
   /**
    * Scrub a dirfrag. It queues child dentries, then does final
    * scrub of the dirfrag.
    *
    * @param dir The dirfrag to scrub (must be auth)
    * @param done set to true if we started to do final scrub
```

Comment:
nit: same as above

Reply:
closing the conversation, this new arg has been removed